dnd-brawler-1/sampleJsonCombatants/direWolf.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "Dire Wold",
+	"name": "Dire Wolf",
```

Comment:
Thanks for spotting this, Can you also fix the spelling error in "crocodile.json" while you're at it ;)

Reply:
...fine. But I'm doing it because I want to do it :P

Reply:
\#manipulated